README.md

```diff
@@ -26,65 +32,52 @@ Extensive documentation is available [here](http://community.optimism.io/docs/)
 * [`ops`](./ops): Contains Dockerfiles for containerizing each service involved in the protocol,
 as well as a docker-compose file for bringing up local testnets easily
 
-## Quickstart
+## Development Quick Start
 
-### Installation
+### Setup
 
-Dependency management is done using `yarn`.
+Clone the repository, open it, and install dependencies:
 
 ```bash
 git clone git@github.com:ethereum-optimism/optimism.git
 cd optimism
-yarn
+yarn install
 ```
 
-After installing the dependencies, you must also build them so that the typescript
-is compiled down to javascript:
+### Building the TypeScript packages
 
-```bash
-yarn build
-```
-
-When changing branches, be sure to clean the repo before building.
+To build all of the [TypeScript packages](./packages), run:
 
 ```bash
 yarn clean
```

Comment:
Any reason to `yarn clean` before anything has ever been built?

Reply:
Just trying to make `yarn clean` part of peoples' habits because it tends to fix conflicts when switching branches.